### PR TITLE
[OSCI] add new author Shatil

### PR DESCRIPTION
**Pull request by ShatilKhan**
### Description
[Add new author Shatil]
 
### Issues Resolved
[None]

### Check List
- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Review by joshuarrrr — CHANGES_REQUESTED** [review]
@ShatilKhan For now, let's remove the photo. Also, please move the new file to the `_authors/osci_2023` directory.

**Comment by dtaivpp** [comment]
@ShatilKhan if you address Josh's comments we can merge this soon! Thank you!

**Comment by ShatilKhan** [comment]
Addressed Guys! @dtaivpp  @joshuarrrr 
Sorry for the delay!

File: _authors/osci_2023/shatil.markdown [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: Shahriar Shatil
+short_name: shatil
+twitter: ShatilKhan2077
+github: ShatilKhan
+---
+
+**Shahriar Shatil** Software developer with experience in React & Node.js , looking to make an impact through his contributions to the OpenSearch project.
```
Comment (joshuarrrr) [inline _authors/osci_2023/shatil.markdown:8]: nit:
```suggestion
**Shahriar Shatil** Software developer with experience in React & Node.js, looking to make an impact through his contributions to the OpenSearch project.
```

**Review by joshuarrrr — APPROVED** [review]
LGTM

**Review by dtaivpp — APPROVED** [review]
Looks good!

**Pull request merged**